CHANGELOG.md

```diff
@@ -176,7 +176,7 @@ for binary in ignite ignited; do
 done
 ```
 
-A more throughout installation guide is available here: https://ignite.readthedocs.io/en/latest/installation.html
+A more throughout installation guide is available here: https://ignite.readthedocs.io/en/latest/installation

```

Comment:
Is there a way to redirect the old link?
It's present in every GitHub release currently:
https://github.com/weaveworks/ignite/releases

Reply:
I don't quite know how to do that. Can we change the release announcements to point to /en/<release>/rest-of-link.html?